tests/bluetooth/host/buf/bt_buf_get_evt/src/test_suite_hci_evt_num_completed_packets.c

```diff
@@ -64,33 +61,44 @@ static struct net_buf_pool *get_memory_pool(bool discardable)
  *     and the same timeout value passed to bt_buf_get_evt()
  *   - bt_buf_get_evt() returns the same reference returned by net_buf_alloc_fixed()
  */
-static void test_returns_not_null_default_events(void)
+ZTEST(bt_buf_get_evt_num_completed_pkts_type_returns_not_null, test_returns_not_null)
```

Comment:
I think you can merge the two suites together, the testcase names are explicit enough